app/models/manageiq/providers/openstack/network_manager/cloud_subnet.rb

```diff
@@ -1,4 +1,20 @@
 class ManageIQ::Providers::Openstack::NetworkManager::CloudSubnet < ::CloudSubnet
+  supports :create_subnet
```

Comment:
I think you can remove the `_subnet` suffix from your features and just use `create`, `update` and `delete`.
Because the instance or class you are asking is already giving you context.
So you would call `subnet.supports_delete?` or `CloudSubnet.class_by_ems(ems).supports_create?`